warehouse/accounts/services.py

```diff
@@ -516,9 +516,7 @@ def __eq__(self, other):
 @implementer(IPasswordBreachedService)
 class HaveIBeenPwnedPasswordBreachedService:
 
-    _failure_message_preamble = (
-        "This password appears in a breach or has been compromised and cannot be used."
-    )
+    _failure_message_preamble = "This password appears in a security breach or has been compromised and cannot be used."
```

Comment:
I tried adding `localize` to this string but when running `make build-mos` I ended up with a lot of conflicts with the locale files. Is there something else I need to do?

Reply:
You don't need to run `make build-mos`, that's what the translation service bot will do. You just need to run `make compile-pot` to add the new translatable string.